### ISSUE TITLE
How to evaluate for my dataset.

### ISSUE THREAD

**Issue by hungnv21292** [davidsandberg/facenet#861]
I have finetune the Asian Face dataset (Asian-Celeb: http://trillionpairs.deepglint.com/overview). beside, I have other my private dataset. (also Asian face). In the original repo of @davidsandberg have evaluate on the LWF use LWF dataset and pairs.txt. How to evaluate on the Asian-Celeb and my private dataset? Do I have to create pairs.txt file as LWF ?

Thank you so much.

**Comment by gitarya** [comment]
Yes, you should align your datasets and generate a pairs.txt on them and then use src/validate_on_lfw.py to evaluate. If you want to generate a pairs.txt, you can use my script here: https://github.com/gitarya/face-data-tools
